app/pages/settings/email.jsx

```diff
@@ -0,0 +1,297 @@
+import React from 'react';
+import apiClient from 'panoptes-client/lib/api-client';
+import talkClient from 'panoptes-client/lib/talk-client';
+import AutoSave from '../../components/auto-save';
+import handleInputChange from '../../lib/handle-input-change';
+
+class EmailSettingsPage extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      page: 1,
+      projects: [],
+      projectPreferences: [],
+      talkPreferences: []
+    };
+    this.handleProjectPreferenceChange = this.handleProjectPreferenceChange.bind(this);
+    this.handleTalkPreferenceChange = this.handleTalkPreferenceChange.bind(this);
+    this.talkPreferenceOption = this.talkPreferenceOption.bind(this);
```

Comment:
I don't think this line is necessary. Only event handlers need to be bound in the constructor. `this.talkPreferenceOption` is a React component.

Reply:
Sorry, missed that.